ethereumj-core/src/main/java/org/ethereum/vm/program/Program.java

```diff
@@ -789,10 +791,29 @@ private int getReturnDataBufferSizeI() {
                 Arrays.copyOfRange(returnDataBuffer, off.intValueSafe(), off.intValueSafe() + size.intValueSafe());
     }
 
+    @Deprecated
```

Comment:
I'd not deprecate `storageLoad` cause `SLOAD` op should keep using it. `getCurrentValue` and `storageLoad` are different operations in terms of semantics, I'd decouple them from each other.